site/versioning-policy.html

```diff
@@ -261,15 +261,15 @@ <h3>Spark 3.4 release window</h3>
   </thead>
   <tbody>
     <tr>
-      <td>January 16th 2023</td>
+      <td>July 16th 2023</td>
```

Comment:
This feels too soon? 3.4 was released a month ago. This schedule should be at least planned for +2 months later than this says, I think

Reply:
Actually I think this was discussed at https://lists.apache.org/thread/l6tprb580lrq1118c83kt9fst30jp8n3 . 3.4 was a bit delayed .. but might be good to stick to the plan ..(?)

Reply:
OK, not against it. The real cadence seems like it's 8 months, though, we could still start the process 'on schedule' and just plan for it taking longer. We'll have a release here with 4 months or so of changes. If the next one slips, we might see releases with just 3 or 2 months. 6 months might be just ambitious

Reply:
I'm good with this PR.

BTW, it seems that we are going to start 3.5.0 and 3.4.1 release at the same time.

Reply:
By convention, do we avoid releasing two versions simultaneously?

Reply:
No~ @xinrong-meng  It's not an issue at all.

Reply:
Thanks @dongjoon-hyun !

Given that we proposed to establish a predictable release cadence early this year, it may be beneficial to stick to that plan for the next release. However, if 3.5.0 experiences further delays, it could be a strong indication that we should reconsider the feasibility of the current release cadence and propose a less ambitious schedule going forward. WDYT @srowen

Reply:
Yeah OK. Another argument is that with only a few months of changes that the release process should hit fewer surprises

Reply:
Totally agree! :)

Reply:
Ya, that's really true.